### ISSUE TITLE
v3.3.3 unable to open on an M1 / Big Sur mac

### ISSUE THREAD

**Issue by SubtleCo** [daskeyboard/daskeyboard.io#110]
From URL: https://www.daskeyboard.io/get-started/software.md

I get the normal / expected "This app cannot be checked for malicious software" but the "open anyway" button just pops up the same alert. I'm blocked!

## System
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro17,1
  Chip:	Apple M1
  Total Number of Cores:	8 (4 performance and 4 efficiency)
  Memory:	16 GB
  System Firmware Version:	6723.140.2
  OS Loader Version:	6723.140.2
<img width="250" alt="Screen Shot 2021-10-06 at 10 50 01 AM" src="https://user-images.githubusercontent.com/62445219/136238829-bfc2d10c-b98c-48df-9ca2-92b1edf47192.png">

**Comment by Mubramaj** [comment]
Have you tried to go to 'System Preferences' -> 'Security & Privacy' then allow apps downloaded from : "App Store and identified developers"

**Comment by SubtleCo** [comment]
Yup!
<img width="599" alt="Screen Shot 2021-10-06 at 11 04 55 AM" src="https://user-images.githubusercontent.com/62445219/136241395-b95300d4-d2de-4144-a8dc-cecd35769bdb.png">

**Comment by Mubramaj** [comment]
If I understand correctly even the 'Open Anyway' button in your screenshot does not open it ? 
Can you go to your download folder in the Finder and try clicking on it again ? (That's what I had to do on my Mac as well)

**Comment by SubtleCo** [comment]
Correct. Clicking the "Open Anyway" button just triggers the modal again

**Comment by Mubramaj** [comment]
Can you try https://www.lifewire.com/fix-developer-cannot-be-verified-error-5183898

By using Safari

**Comment by SubtleCo** [comment]
No dice.

Deleted DMG, restarted, downloaded from Safari, same issue.

**Comment by SubtleCo** [comment]
This is still an issue - Is anyone else on an M1 running MacOS 11.6 able to replicate?

**Comment by mugizico** [comment]
Not on an M1 but running into a very similar issue: https://github.com/daskeyboard/daskeyboard.io/issues/113 I just got the keyboard today. (5QS)

**Comment by freddiehaddad** [comment]
Right-click on the package from the Finder window and click open.  When you are prompted with about verification, you should see an option to click open.  If you double-click on the package, you will NOT see that option.
